charts/alfresco-connector-msteams/templates/secret-msteams.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if not .Values.microsoft.app.existingSecret.name }}
+apiVersion: v1
+kind: Secret
+metadata:
+  {{- $secretCtx := dict "Values" (dict "nameOverride" "msteams-se") "Chart" .Chart "Release" .Release }}
+  name: {{ template "alfresco-connector-msteams.fullname" $secretCtx }}
```

Comment:
it's better to have a named helper so you avoid this copy-pasta in multiple places, like I did [here](https://github.com/Alfresco/alfresco-helm-charts/pull/175/files#diff-cead3e4fd48780ae4d8f8e2cc4fb668b1e73bc40a836a1a143c13170e36b98fa)